lib/googleanalytics.js

```diff
@@ -21,15 +24,70 @@ module.exports = {
     ga('create', process.env.GA_ACCOUNT, 'auto');
     ga('send', 'pageview');
   },
+
   /**
    * pageview:
    * a proxy to basic GA pageview tracking
-   * @param  {[type]} path [description]
-   * @return {[type]}      [description]
+   * @param  {String} path - the current page page e.g. '/about'
    */
   pageview: function (path) {
     if (typeof ga === 'function') {
       ga('send', 'pageview', path);
     }
+  },
+
+  /**
+   * modalview:
+   * a proxy to basic GA pageview tracking
+   * @param  {String} modalName e.g. 'add-or-edit-club'
+   */
+  modalview: function (modalName) {
+    path = '/modal/' + modalName;
```

Comment:
If we decide to make the modals visible at their own URLs so that e.g. the "add your club" modal is visible at "/mozilla-web-clubs/add/", is this solution compatible with that?